File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Java/src/test/java/simulacionBandaMusica/factories/CantanteProFactoryTest.java
package simulacionBandaMusica.factories;

import org.junit.jupiter.api.Test;
import simulacionBandaMusica.models.CantantePro;

import static org.junit.jupiter.api.Assertions.assertNotNull;

class CantanteProFactoryTest {

    @Test
    void createCantanteProTest() {
        CantantePro CantantePro = CantanteProFactory.create();
        // CantantePro no nulo, para ver si se han asignado correctamente los datos
        assertNotNull(CantantePro);
    }
}

File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Kotlin/src/test/kotlin/simulacionBandaMusica/factories/CantanteProFactoryTest.kt
package simulacionBandaMusica.factories

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import simulacionBandaMusica.models.Musico

internal class CantanteProFactoryTest {
    @Test
    fun createCantanteProTest() {
        val CantantePro: Musico.CantantePro = CantanteProFactory.create()
        // CantantePro no nulo, para ver si se han asignado correctamente los datos
        Assertions.assertNotNull(CantantePro)
    }
}